db/migrate/20230117180357_create_tulsa_city_inmates.rb

```diff
@@ -0,0 +1,28 @@
+class CreateTulsaCityInmates < ActiveRecord::Migration[6.0]
+  def change
+    create_table :tulsa_city_inmates do |t|
+      t.string :inmateId, index: {unique: true}
```

Comment:
good job on unique indexes
Column names are always snake case in ruby e.g., `inmate_id`